recipes/cubicinterpolation/all/conanfile.py

```diff
@@ -0,0 +1,56 @@
+from conans import ConanFile, CMake, tools
+import glob
+import os
+
+class CubicInterpolationConan(ConanFile):
+    name = "CubicInterpolation"
+    homepage = "https://github.com/MaxSac/cubic_interpolation"
+    version = "0.1"
+    license = "MIT"
+    author = "Maximilian Sackel mail@maxsac.de"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Leightweight interpolation library based on boost and eigen."
+    topics = ("interpolation", "splines", "cubic", "bicubic", "boost", "eigen3")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake_find_package"
+    _source_folder = "source_folder"
+    exports_sources = "*"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob("cubic_interpolation-*/")[0]
```

Comment:
```suggestion
        extracted_dir = "cubic_interpolation-" + self.version
```
Don't need glob if you know the version